server/src/test/java/org/elasticsearch/common/bytes/CompositeBytesReferenceTests.java

```diff
@@ -163,4 +163,18 @@ public void testGetIntLE() {
         Exception e = expectThrows(ArrayIndexOutOfBoundsException.class, () -> comp.getIntLE(5));
         assertThat(e.getMessage(), equalTo("Index 4 out of bounds for length 4"));
     }
+
+    public void testGetDoubleLE() {
+        // first bytes array = 1.2, second bytes array = 1.4, third bytes array = 1.6
+        BytesReference[] refs = new BytesReference[] {
+            new BytesArray(new byte[] { 0x33, 0x33, 0x33, 0x33, 0x33, 0x33, -0xD, 0x3F }),
```

Comment:
Maybe we should make one big byte array and then randomly break it into smaller arrays?

Reply:
pushed: 7f9f0840a5cd3f1882e62b097ac8e0d64e0b6932